src/utils.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use error_chain::bail;
+use std::process::Command;
+
+use crate::errors::*;
+
+#[macro_export]
+macro_rules! runcmd {
+    ($cmd:expr, $($args:expr),*) => {{
+        let mut cmd = Command::new($cmd);
+        $( cmd.arg($args); )*
+        // print full cmdline tried here?
```

Comment:
I think that would be better, yes.  `Command`'s `Debug` implementation does exactly this.

Reply:
Oh nice, missed that.